content/integrations/btrfs.md

```diff
@@ -5,14 +5,49 @@ kind: integration
 doclevel: basic
 git_integration_title: btrfs
 ---
+{{< img src="integrations/btrfs/btrfs_metric.png" >}}
+
+## Overview 
 
 Capture Btrfs metrics into Datadog to:
 
 * Visualize your file system performance.
 * Correlate the performance of Btrfs file system with the rest of your applications.
 
+## Installation 
+The Btrfs check is packaged with the Agent, so simply [install the Agent](https://app.datadoghq.com/account/settings#agent) on every server that uses at least one Btrfs filesystem.
+
+## Configuration 
+To capture Btrfs metrics you need to install the Datadog Agent:
```

Comment:
Installing the Agent is already (and always is) part of the **Installation** section.